src/config.c

```diff
@@ -2218,6 +2218,21 @@ static void numericConfigRewrite(standardConfig *config, const char *name, struc
     }
 }
 
+sds getConfigValue(sds config_name) {
+    standardConfig *config = lookupConfig(config_name);
+    switch (config->type) {
+    case BOOL_CONFIG:
+        return boolConfigGet(config);
+    case SDS_CONFIG:
+        return sdsConfigGet(config);
+    case NUMERIC_CONFIG:
+        return numericConfigGet(config);
+    case ENUM_CONFIG:
+        return enumConfigGet(config);
+    default: serverPanic("Config type of module config is not allowed.");
```

Comment:
we don't support the other type? like STRING_CONFIG or SPECIAL_CONFIG, i guess SPECIAL_CONFIG is a special one, but STRING_CONFIG?

Reply:
STRING_CONFIG is not supported so far by origin design, I have no idea the reason.